### PR TITLE
replace link to yari-demos.prod.mdn.mozit.cloud with LiveSampleLink macro

### LINKED ISSUE #34326: Remove 2 links to yari-demos.prod.mdn.mozit.cloud
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event#window_size_logger

### What specific section or headline is this issue about?

Window size logger

### What information was incorrect, unhelpful, or incomplete?

There are two links to `yari-demos.prod.mdn.mozit.cloud`, which is now decommissioned: https://github.com/search?q=org%3Amdn+yari-demos+repo%3Amdn%2Fcontent&type=code

The other occurence is in the Writing guidelines and suggests to use these links:  https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Live_samples#grouping_code_blocks

### What did you expect to see?

The links may be replaced by something like this: https://live.mdnplay.dev/en-US/docs/MDN/Writing_guidelines/Page_structures/Live_samples/_sample_.grouping_code_blocks_by_heading.html?id=grouping_code_blocks_by_heading&amp;url=%2Fen-US%2Fdocs%2FMDN%2FWriting_guidelines%2FPage_structures%2FLive_samples

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
The window size logger link should be replaced with the LiveSampleLink macro

### Related issues and pull requests

https://github.com/mdn/content/issues/34326

### CODE REVIEW

File: files/en-us/web/api/window/resize_event/index.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ window.onresize = reportWindowSize;
 
 > **Note:** The example output here is in an {{HTMLElement("iframe")}}, so the reported width and height values are for the `<iframe>`, not the window that this page is in. In particular, it will be hard to adjust the window size so as to see a difference in the reported height.
 >
-> The effect is easier to see if you [view the example in its own window](https://yari-demos.prod.mdn.mozit.cloud/en-US/docs/Web/API/Window/resize_event/_sample_.window_size_logger.html).
+> The effect is easier to see if you {{LiveSampleLink('Window_size_logger', "view the example in its own window")}}.
```
Comment (Josh-Cena) [inline files/en-us/web/api/window/resize_event/index.md:69]:
```suggestion
> The effect is easier to see if you {{LiveSampleLink("Window_size_logger", "view the example in its own window")}}.
```

Why change to single quotes?

**Review by Josh-Cena — APPROVED** [review]
Great work 👍

File: files/en-us/web/api/window/resize_event/index.md (hunk 1 above)
Comment (gagan-bhullar-tech) [reply files/en-us/web/api/window/resize_event/index.md]: updated to double quotes

**Comment by Josh-Cena** [comment]
@gagan-bhullar-tech Do you want to fix the other mention in https://github.com/mdn/content/issues/34326 too?

**Pull request merged**